proto/cosmos/distribution/v1beta1/distribution.proto

```diff
@@ -110,8 +110,8 @@ message CommunityPoolSpendProposal {
 
   string   title                           = 1;
   string   description                     = 2;
-  string   recipient                       = 3;
-  repeated cosmos.base.v1beta1.Coin amount = 4
+  string   recipient                       = 4;
+  repeated cosmos.base.v1beta1.Coin amount = 5
```

Comment:
I think you meant to undo these changes?

Reply:
Sorry, missed some of them. Will do a better job double-checking this time